src/hooks/sales.ts

```diff
@@ -293,10 +293,17 @@ const GetSalesLeadMemberData = gql`
     }
   }
 `
+
 const GetSalesLeadMembers = gql`
   query GetSalesLeadMembers($appId: String!, $managerId: String!) {
     member(
-      where: { app_id: { _eq: $appId }, manager_id: { _eq: $managerId }, member_phones: { phone: { _is_null: false } } }
+      where: {
+        app_id: { _eq: $appId }
+        manager_id: { _eq: $managerId }
+        member_phones: { phone: { _is_null: false } }
+        excluded_at: { _is_null: true }
+        _or: [{ star: { _neq: -9999 } }, { star: { _is_null: true } }]
```

Comment:
could be  `{  star: { _gte: -9999} }`

Reply:
don't fix here. it should fix frontend component part